keps/sig-scheduling/1258-default-pod-topology-spread/README.md

```diff
@@ -259,13 +272,200 @@ To ensure this feature to be rolled out in high quality. Following tests are man
+
+  Since spreading is affected, node utilization might change.
+  Utilization metrics can be queried in the `/metrics/resource` endpoint exposed by kubelet.
+
+* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+
+  N/A.
+
+* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
+  fields of API types, flags, etc.?**
+
+  TBD for GA.
+
+### Monitoring Requirements
+
+* **How can an operator determine if the feature is in use by workloads?**
+
+  All Pods are affected, unless they have explicit spreading constraints (.spec.topologySpreadConstraints).
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine 
+  the health of the service?**
+
+  - [x] Metrics
+    - Metric name: `framework_extension_point_duration_seconds` with label `extension_point` values `PreScore` and/or `Score`.
+    - [Optional] Aggregation method:
+    - Components exposing the metric: `kube_scheduler`.
+  - [ ] Other (treat as last resort)
+    - Details:
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+
+  For 100 nodes, with a 4-core master:
+
+  - Latency for PreScore less than 15ms for 99% percentile.
+  - Latency for Score less than 50ms for 99% percentile.
+
+* **Are there any missing metrics that would be useful to have to improve observability 
+of this feature?**
+
+  N/A.
+
+### Dependencies
+
+* **Does this feature depend on any specific services running in the cluster?**
+
+  N/A.
+
+
+### Scalability
+
+* **Will enabling / using this feature result in any new API calls?**
+
+  No.
+
+* **Will enabling / using this feature result in introducing new API types?**
+
+  No.
+
+* **Will enabling / using this feature result in any new calls to the cloud 
+provider?**
+
+  No.
+
+* **Will enabling / using this feature result in increasing size or count of 
+the existing API objects?**
+
+  No.
+
+* **Will enabling / using this feature result in increasing time taken by any 
+operations covered by [existing SLIs/SLOs]?**
+
+  Scheduling time on clusters with more than 100 nodes. Smaller clusters are unaffected.
+  `SelectorSpreading` doesn't take into account all the Nodes in big clusters when calculating skew,
+  resulting in partial spreading at this scale.
+  On the contrary, `PodTopologySpreading` considers all nodes when using topologies bigger than a Node, like a Zone.
+
+* **Will enabling / using this feature result in non-negligible increase of 
+resource usage (CPU, RAM, disk, IO, ...) in any components?**
+
+  kube-scheduler needs to use more CPU to calculate Zone spreading.
+
+### Troubleshooting
+
+* **How does this feature react if the API server and/or etcd is unavailable?**
+
+  kube-scheduler won't receive Pods
+  The effect is no more than it be without the feature.
+
+* **What are other known failure modes?**
+
+  - Pod scheduling is slow
+    - Detection: Pod creation latency is too high.
```

Comment:
s/creation/startup time/ ?
Or scheduling?

Reply:
startup time. That is the first symptom, and then you narrow down from there :)